app/femr/ui/controllers/admin/InventoryController.java

```diff
@@ -75,7 +75,7 @@ public Result manageGet() {
         CurrentUser currentUser = sessionService.retrieveCurrentUserSession();
 
         InventoryViewModelGet viewModel = new InventoryViewModelGet();
-        ServiceResponse<List<MedicationItem>> medicationServiceResponse = medicationService.retrieveMedicationInventory();
+        ServiceResponse<List<MedicationItem>> medicationServiceResponse = medicationService.retrieveMedicationInventory(currentUser.getTripId());
         if (medicationServiceResponse.hasErrors()) {
```

Comment:
If a user is not assigned to a trip, currentUser.getTripId() is null, which Java does not like. This ties into https://teamfemr.atlassian.net/browse/FEMR-169